src/ansys/dpf/core/misc.py

```diff
@@ -156,6 +161,9 @@ def find_ansys():
     >>> path = find_ansys()
 
     """
+    latest_server = _find_latest_ansys_dpf_server()
```

Comment:
@PProfizi I don't think that it's the expected behavior. We want the latest version through both the standalone and the ansys versions. That's why it was a single function

Reply:
@cbellot000 I think it is. Otherwise, what you got locally and we called a bug, is actually the intended behavior:
- we have ansys-dpf-server-2024.1.pre0 installed
- we have AWP_ROOT241, which is translated to 2024.1.0
- > the AWP_ROOT241 wins

Yet, the behavior above is not what we want IMO.

Reply:
@cbellot000 I agree with what @PProfizi is saying. If a user installs some server version (say 2023.pre.1) in the venv and then 1 year later an Ansys unified install is installed (say 25.2), and the user wants to use this one instead, what he should do is uninstall the server. The problem is how to communicate this (I guess that printing the ansys path to the std output each time we import dpf is not a good thing)

Reply:
@rafacanton and @PProfizi let's discuss this in a meeting